app/javascript/components/ingredients/base-form-component.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="pt-3">
+    <form class="w-full max-w-lg">
+      <div class="flex flex-wrap -mx-3 mb-6">
+        <div class="w-full px-3">
+          <input-form
+            id="ingredient-name"
+            type="text"
+            placeholder="Nombre"
+          />
+        </div>
+      </div>
+
+      <div class="flex flex-wrap -mx-3 mb-6">
+        <div class="w-full md:w-1/2 px-3 mb-6 md:mb-0">
+          <input-form
+            id="ingredient-amount"
+            type="number"
+            placeholder="Cantidad"
+          />
+        </div>
+        <div class="relative">
+          <select
+            class="block appearance-none w-full bg-white border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none"
+            id="ingredient-unit"
+          >
+            <option
+              hidden
+              selected
+            >
+              Unidad
+            </option>
+            <option v-for="unit in units">
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key